File: c#-1/exibirMaiorOuMenorIdade.cs
using System;

class exibirMaiorOuMenorIdade 
{
  public static void Main (string[] args) 
  {
    const int atual = 2022;
    int ano_inserido;
    int idade;
    string nome;

    Console.WriteLine("Digite seu nome");
    nome = Convert.ToString(Console.ReadLine());
    Console.WriteLine("Digite o ano que você nasceu");
    ano_inserido = Convert.ToInt32(Console.ReadLine());

    idade = atual - ano_inserido;

    if(idade >= 18) {
      Console.WriteLine("Você é maior de idade");
    }
    else {
      Console.WriteLine("Você é menor de idade");
    }
  }
}

File: js-lista1/exibirMaiorOuMenorIdade.js
const atual = 2022;

let nome = prompt("Digite seu nome: ");
let ano_inserido = prompt("Digite o ano em que você nasceu: ");

idade = atual - ano_inserido;

if (idade >= 18) {
  console.log("Você é maior de idade");
} else {
  console.log("Você é menor de idade");
}